src/main/java/com/app/registersystem/controller/UserController.java

```diff
@@ -24,12 +25,26 @@
 import com.app.registersystem.service.UserService;
 import com.app.registersystem.validation.DtoValidationUtils;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@Api(value="User controller",description = "Rest Api related to User entity!!")
 @RestController
 @RequestMapping("/user")
 public class UserController {
 	@Autowired
 	private UserService userService;
 
+	
+	@ApiOperation(value="adding person into records")
+	@ApiResponses(value = {
+			@ApiResponse(code = 200,message = "Success"),
```

Comment:
I think These are default messages.We can remove the messages or we can have other messages regarding that response code